components/alert/alert.css

```diff
@@ -0,0 +1,80 @@
+@import url('/index.css');
+
+
+.alert-bg {
+    background-color: #fff;
+}
+
+.alert-container {
+    width: 70%;
+    border: 2px solid gray;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    margin: auto;
+    height: 22rem;
+    border-radius: 5px;
+    background-color: var(--bg-color);
+}
+
+.alert {
+    padding: 1rem 2rem 1rem 0.7rem;
+    margin: 0.5rem;
+    border-radius: 5px;
+    color: #fff;
+    width: 80%;
+}
+
+.information {
+    background-color: rgb(99, 181, 231);
+}
+
+.success {
+    background-color: rgba(38, 189, 18, 0.726);
+}
+
+.error {
+    background-color: rgb(248, 64, 64);
+}
```

Comment:
Use same color type throughout the code , If you are  using hex , use it eveywhere